fitbenchmarking/fitbenchmarking
# Core tests bug

#### Description of Work

Fixes #675 

Fixes tests for both `fitting_benchmarking.benchmark` and `results_output` modules.

#### Testing Instructions

1. Check CI tests pass

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
